Dockerfile

```diff
@@ -36,7 +36,7 @@ ADD docker/start.sh /app/
 ADD . /app
 
 RUN useradd -m stringer
-RUN chown -R stringer:stringer /app
+RUN chown -R stringer:stringer /app /etc/supervisord.conf
```

Comment:
Can you tell me more about why this was necessary?

Reply:
Otherwise the supervisord is failing with error "Cannot read /etc/supervisord.conf"